src/SidebarProvider.ts

```diff
@@ -46,8 +47,38 @@ export class SidebarProvider implements vscode.WebviewViewProvider {
             vscode.commands.executeCommand(
               "workbench.files.action.showActiveFileInExplorer"
             );
+
+            // Run code to automatically install files in a folder
+            const workspace = vscode.workspace.workspaceFolders?.[0];
+            const currentWorkspacePath = workspace.uri.fsPath;
+            if (!process.env.RUN_COMMAND){
```

Comment:
I presume right now we only run one command taken from env called `RUN_COMMAND` in the main workspace root. Let me know if this is okay

Reply:
would we need a separate install command first?

Reply:
Actually this should probably be called `INSTALL_COMMAND` for better accuracy 🤔 Do we want to start a RUN_COMMAND as well then?